install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -263,6 +263,15 @@ containers:
   - name: STACKDRIVER_TRACING_MAX_NUMBER_OF_MESSAGE_EVENTS
     value: "{{ .ProxyConfig.GetTracing.GetStackdriver.GetMaxNumberOfMessageEvents }}"
   {{- end }}
+  {{ $envs := .Values.global.proxy.env }}
```

Comment:
@rshriram I think the test case you're mentioning is covered here: https://github.com/istio/istio/pull/18333/files#diff-f952abe8a9568c4b6f7d8eef30eb7114R103

or am I missing something? I'm assuming `global.proxy` will always exist because of the current structure of the `values.yaml` file.. is this assumption too optimistic?